hs_discover/package.json

```diff
@@ -39,7 +40,14 @@
       "plugin:vue/essential",
       "@vue/airbnb"
     ],
-    "rules": {},
+    "rules": {
+      "max-len": [
+        "error",
+        {
+          "code": 255
+        }
```

Comment:
Line length extended since things got too disjointed without

Reply:
These rules work for command-line, PyCharm or other IDE and should be linked into any automated CI/CD linting/static code analysis configuration for consistence and single-point of control